test/apps/iodemo/io_demo.cc

```diff
@@ -603,7 +603,7 @@ class P2pDemoCommon : public UcxContext {
     public:
         BufferIov(size_t size, MemoryPool<BufferIov> &pool) :
                 _data_size(0lu), _memory_type(UCS_MEMORY_TYPE_UNKNOWN),
-                _validate(false), _pool(pool), _extra_buf(NULL)
+                _validate(false), _pool(pool), _extra_buf(NULL), _bogus_buf(NULL)
```

Comment:
does it fit the "80 symbols per line" rule?

Reply:
done